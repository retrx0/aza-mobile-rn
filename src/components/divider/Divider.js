"use strict";
exports.__esModule = true;
var Colors_1 = require("../../constants/Colors");
var useColorScheme_1 = require("../../hooks/useColorScheme");
var Themed_1 = require("../Themed");
var Divider = function () {
    var colorScheme = useColorScheme_1["default"]();
    return (<Themed_1.View style={{
            backgroundColor: "transparent",
            borderBottomWidth: 0.6,
            borderBottomColor: Colors_1["default"][colorScheme].separator
        }}/>);
};
exports["default"] = Divider;
