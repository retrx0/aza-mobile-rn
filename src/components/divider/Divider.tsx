import Colors from "../../constants/Colors";
import useColorScheme from "../../hooks/useColorScheme";
import { View } from "../Themed";

const Divider = () => {
  const colorScheme = useColorScheme();

  return (
    <View
      style={{
        backgroundColor: "transparent",
        borderBottomWidth: 0.6,
        borderBottomColor: Colors[colorScheme].separator,
      }}
    />
  );
};

export default Divider;
