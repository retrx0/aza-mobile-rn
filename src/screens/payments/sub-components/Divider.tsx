import { View, Text,StyleSheet } from 'react-native'
import React from 'react'

export default function Divider() {
  return (
    <View style={styles.divide}></View>
  )
}
const styles = StyleSheet.create({
divide:{
    width:'100%',
    height:1,
    backgroundColor:'#EAEAEC',
    marginTop:20
}
})