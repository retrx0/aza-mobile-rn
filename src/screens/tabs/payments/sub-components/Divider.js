"use strict";
exports.__esModule = true;
var react_native_1 = require("react-native");
var react_1 = require("react");
var useColorScheme_1 = require("../../../../hooks/useColorScheme");
var Colors_1 = require("../../../../constants/Colors");
function Divider(_a) {
    var style = _a.style;
    var colorScheme = useColorScheme_1["default"]();
    return (<react_native_1.View style={[
            styles.divider,
            style,
            {
                backgroundColor: Colors_1["default"][colorScheme].disabledButton
            },
        ]}></react_native_1.View>);
}
exports["default"] = Divider;
var styles = react_native_1.StyleSheet.create({
    divider: {
        width: "100%",
        height: 0.5,
        marginTop: 20,
        marginLeft: "auto",
        marginRight: "auto"
    }
});
