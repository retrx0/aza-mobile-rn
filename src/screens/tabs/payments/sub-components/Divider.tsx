import { View, StyleSheet, ViewStyle } from "react-native";
import React from "react";
import useColorScheme from "../../../../hooks/useColorScheme";
import Colors from "../../../../constants/Colors";

export default function Divider({ style }: { style?: ViewStyle }) {
  const colorScheme = useColorScheme();

  return (
    <View
      style={[
        styles.divider,
        style,
        {
          backgroundColor: Colors[colorScheme].disabledButton,
        },
      ]}></View>
  );
}
const styles = StyleSheet.create({
  divider: {
    width: "100%",
    height: 0.5,
    marginTop: 20,
    marginLeft: "auto",
    marginRight: "auto",
  },
});
