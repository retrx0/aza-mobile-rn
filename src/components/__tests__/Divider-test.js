import * as React from "react";
import renderer from "react-test-renderer";
import Divider from "../divider/Divider";

it(`renders divider correctly`, () => {
  const tree = renderer.create(<Divider />).toJSON();

  expect(tree).toMatchSnapshot();
});
