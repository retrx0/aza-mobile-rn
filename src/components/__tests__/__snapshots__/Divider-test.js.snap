// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders divider correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "borderBottomColor": "#EAEAEC",
      "borderBottomWidth": 0.6,
    }
  }
/>
`;
